server/rest/src/main/java/org/infinispan/rest/resources/ContainerResource.java

```diff
@@ -578,6 +610,19 @@ public EventStream getEventStream() {
          return eventStream;
       }
 
+      @CacheEntryCreated
+      public CompletionStage<Void> onEntryCreated(CacheEntryCreatedEvent<?, EventLog> entry) {
```

Comment:
As the cache always has `UUID` as a key, I think we can replace the wildcard with `UUID` and avoid the unchecked cast.